src/main/cs/Infinispan/HotRod/Config/ConfigurationBuilder.cs

```diff
@@ -85,6 +85,11 @@ public SslConfigurationBuilder Ssl()
             return new SslConfigurationBuilder(this, builder.Ssl());
         }
 
+        public SecurityConfigurationBuilder Security()
```

Comment:
So now we have Security at the same level as Ssl(). The java client has it differently: Security at the top level, and Authentication and Ssl under it. Ssl should probably be under Security.